charts/redpanda/templates/secrets.yaml

```diff
@@ -257,3 +263,95 @@ stringData:
     sleep infinity
   {{- end }}
 {{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ (include "redpanda.name" .) | trunc 51 }}-configurator
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+  {{- with include "full.labels" . }}
+  {{- . | nindent 4 }}
+  {{- end }}
+type: Opaque
+stringData:
+  configurator.sh: |-
+    # TODO you have to bring back the range so that the external address list is ammended properly
+    # once that is done, by bringing back the range here, you then must fine a way to only pick up the listeners that pertain
+    # to the ordinal we are working with, cutting should be the right approach here
```

Comment:
Is this a note to self, or is this something that still needs done. If the latter, what do we get or miss by not doing it now?

Reply:
will remove